microservices-connector/internal/controller/suite_test.go

```diff
@@ -55,12 +82,44 @@ var _ = BeforeSuite(func() {
 			fmt.Sprintf("1.29.0-%s-%s", runtime.GOOS, runtime.GOARCH)),
 	}
 
-	var err error
 	// cfg is defined in this file globally.
-	cfg, err = testEnv.Start()
+	cfg, err := testEnv.Start()
+
 	Expect(err).NotTo(HaveOccurred())
 	Expect(cfg).NotTo(BeNil())
 
+	kubeconfigContent := fmt.Sprintf(`
```

Comment:
Use fakeclient instead

Reply:
the whole test is running on a fake environment and schema, I tried to use fakeClient, it does not work instead, cannot Get/Create resource